install.sh

```diff
@@ -1,42 +1,57 @@
 #!/bin/bash -e
-set -u
+
+OLM_VERSION=0.17.0
+OPERATOR_INDEX_IMAGE=${OPERATOR_INDEX_IMAGE:-registry.redhat.io/redhat/community-operator-index:latest}
+OPERATOR_REGISTRY_REF=$(echo $OPERATOR_INDEX_IMAGE | cut -f 1 -d '/')
+OPERATOR_CHANNEL=${OPERATOR_CHANNEL:-beta}
+DOCKER_CFG=${DOCKER_CFG:-$HOME/.docker/config.json}
+CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-docker}
 
 #The SBO can also be installed in a vanilla kubernetes cluster. A prerequisite for this would be to add credentials for the registry.redhat.io in the cluster. Steps to be followed to achieve the same are listed below:
 #1) Follow [Red Hat Container Registry Authentication Steps](https://access.redhat.com/RegistryAuthentication)
 #2) Verify that your credentials are correct using docker login -u <your_username> -p <your_passwd> registry.redhat.io
-docker login registry.redhat.io
+
+if [ -z "$SKIP_REGISTRY_LOGIN" ]; then
```

Comment:
When would this/can be false?

Reply:
* when a registry does not require authentication or
* when the environment already is configured (authenticated before) so that we do not need to repeat it.

Reply:
When `$SKIP_REGISTRY_LOGIN` is set to a any non-empty value.